src/spectacle-theme.js

```diff
@@ -276,12 +276,12 @@ export default {
   /*
    * Ecology text wrangling
    */
-  ".Overview pre": {
+  ".Ecology pre": {
```

Comment:
The ecology component had `.Ecology` as a wrapper class and `.Overview` class for the API section (and a separate class for the component playgrounds and props tables, if I recall correctly). 

The `<Docs />` component heavily relies on the `.Ecology` class, but I could rename all instances of it in this theme file.